### DISCUSSION TITLE
如何解决元数据备份inodes太多警告问题

### DISCUSSION THREAD

**Discussion by github-zl910** [juicedata/juicefs#4295] [Q&A]
warning:backup metadata skipped because of too many inodes:1340401 1h0m0s;you may increase '--backup --meta' to enable it again. 我在重新挂载时设置备份时隔为8h，但是仍提示上述内容，我的设置juicefs mount -d --backup-meta 8h "mysql://juicefs:xxx/juicefs" /opt/jfs，请问有人能帮我看依稀为何不生效吗？

**Comment by zhijian-pro** [comment]
What is the version ?
Can you take a screenshot of the original command ?
